site/content/docs/howto/debugging.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Debugging Pinniped 
+description: Debugging `pinniped` configuration errors.
+description: 
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Debugging
+    weight: 900
+    parent: howtos
+---
+
+## Debugging on the Client 
+
+The `PINNIPED_DEBUG=true` environment variable can be set to enable additional CLI logging.
+
+## Debugging on the Server
+
+To adjust the log level of either the Pinniped Supervisor or the Pinniped Concierge the log level value must be updated 
+in the appropriate configmap associated with each deployment. 
+
+The `log level` options are as follows: 
+- `info` ("nice to know" information) 
+- `debug` (developer information) 
+- `trace` (timing information)
+- `all` (kitchen sink)
+
+Do not use `trace` or `all` on production systems, as credentials may get logged. When this value is left unset, 
+only warnings and errors are printed. There is no way to suppress warning and error logs.
+
+Choose an update method that corresponds with the original installation method you chose for your cluster. Consult
+[Install Supervisor]({{< ref "../howto/install-supervisor" >}}) or [Install Concierge]({{< ref "../howto/install-concierge" >}})
+for more information.
+
+### Using kapp
+
+To adjust the log level of the Pinniped Supervisor or Concierge using `kapp`, edit the `log_level` in the `values.yaml` files 
+in your local copy of the Pinniped GitHub repository. Open either `deploy/supervisor/values.yaml` or `deploy/concierge/values.yaml` 
+and edit the following line: 
+
+```yaml
+log_level: "info|debug|trace|all"
+```
+Then apply your configuration via `kapp deploy -f`.
+
+### Using kubectl
+
+To adjust the log level of the Pinniped Supervisor or Concierge using `kubectl`, find the configmap on your cluster within
+the namespace of each deployment:
+
+```bash
+# get the concierge config
+kubectl get cm pinniped-concierge-config --namespace concierge --output yaml > cm.concierge.yaml
```

Comment:
Does this have the word `static` in its name, or is it really different from the supervisor?

Reply:
It does have the word `-static` in the name:

![Uploading Screenshot 2024-03-27 at 5.28.05 PM.png…]()

They are unfortunately slightly different in name.